lua/config/defaults.lua

```diff
@@ -20,7 +20,7 @@ lvim = {
     level = "warn",
     viewer = {
       ---@usage this will fallback on "less +F" if not found
-      cmd = "lnav",
+      cmd = "less",
```

Comment:
This is not the correct place to change it. 

There's supposed to be a fallback in `toggle_log_viewer` but it disappeared somehow. It should also be `less +F` or `tail -f`.